client/src/views/navbar-menu.js

```diff
@@ -1,22 +1,109 @@
 import Snabbdom from 'snabbdom-pragma'
 import { updateQuery } from '../util'
+import { toggleIcon } from './util'
 
 const items  = process.env.MENU_ITEMS && JSON.parse(process.env.MENU_ITEMS)
     , active = process.env.MENU_ACTIVE
 
 const otherTheme = { dark: 'light', light: 'dark' }
+const staticRoot = process.env.STATIC_ROOT || ''
 
 export default ({ t, theme, page }) =>
 
-  <ul className="navbar-nav">
+<div className="main-nav-container">
+  <ul className="main-nav">
     { items && Object.entries(items).map(([ name, url ]) =>
         <li className={`nav-item ${name == active ? 'active' : ''}`}>
-          <a className="nav-link" href={url} rel="external">{t(name)}</a>
+          <a className="nav-link" href={url} rel="external">
+            <span><img className="menu-logo" alt="" src={`${staticRoot}img/icons/menu-logo.png`} /></span>
+            <span>{t(name)}</span>
+          </a>
         </li>
     ) }
-    <li className="nav-item nav-toggler">
-      { process.browser ? <div className="nav-bar_moon_icon toggle-theme"></div>
-                        : <a href={page.pathname.substr(1) + updateQuery(page.query, { theme: otherTheme[theme] })} className="nav-bar_moon_icon"></a>
-      }
-    </li>
   </ul>
+    
+  <div className="toggle-container">
+    <div className="burger-icon">
+        <span></span>
+        <span></span>
+        <span></span>
+    </div>
+    {toggleIcon()}
+    <div className="toggle-menu">
+      <div className="toggle-menu-header">
+        <div className="switch-theme-icon"></div>
+        <div className="language">
+          <form method="get">
+            { !process.browser && Object.entries(page.query).map(([k, v]) =>
+              k != 'lang' && <input type="hidden" name={k} value={v} />
+            ) }
+            <select className="language-selector" name="lang">
+              { Object.entries(t.langs).map(([ lang_id, lang_t ]) =>
+                <option value={lang_id} attrs={lang_id == t.lang_id ? { selected: true } : {}}>{lang_t`lang_name`}</option>
+              ) }
+            </select>
+            { !process.browser && <input type="submit" className="language-submit" value={t`Go`} /> }
+          </form>
+        </div>
+      </div>
+      <div className="section1">
+        <h4 className="menu-title">Wallets</h4>
```

Comment:
So far, we kept all the Blockstream-specific stuff (logo/branding, copyright notice, social media links, meta description, etc) separate from the mainline code (inside the "blockstream" flavor directory), so that its not included for folks that self-host their own esplora instances. I think that it would great if we could keep it that way. :)

This can be done here by moving this HTML chunk into an envirnoment variable set by `flavors/blockstream/config.env`, similarly to `HEAD_HTML`.

Reply:
This is also relevant for the footer ToS/privacy links.

Reply:
OK. I'll add Privacy ToS/privacy links to the config.

Reply:
I've added Terms and Privacy to the config